example/actions/move_to_pos.lua

```diff
@@ -5,8 +5,16 @@ local M = {
     type = 'Action',
     desc = '移动到坐标',
     args = {
-        {'x', 'int', 'x'},
-        {'y', 'int', 'y'}
+        {
+            name = 'x',
+            type = 'int',
+            desc = 'x坐标'
+        },
+        {
+            name = 'y',
+            type = 'int',
+            desc = 'y坐标'
+        }
```

Comment:
This the proper way to pass args according to generic nodes https://github.com/zhandouxiaojiji/behavior3lua/blob/master/behavior3/nodes/conditions/cmp.lua#L14-L21